src/styles/main.scss

```
@@ -1,3 +1,235 @@
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-sat:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  @for $number from 1 through 30 {
+    &__day-start-day-sun:nth-child(#{$number})::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: #000;
+    }
+  }
+
+  &__day {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  &__day-start-day-mon {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  &__day-start-day-tue {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  &__day-start-day-wed {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  &__day-start-day-thu {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  &__day-start-day-fri {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  &__day-start-day-sat {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  &__day-start-day-sun {
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 98px;
+    height: 98px;
+
+    transition: color 0.5, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
```

Add month-length modifier
```suggestion
@for $month-day from 28 through 31 {
    &--month--lenth-#{$month-day} &__day:nth-child(n + #{$month-day + 1}) {
      display: none;
    }
  }
```